numpy/random/tests/test_generator_mt19937.py

```diff
@@ -2349,6 +2343,61 @@ def test_multinomial(self):
                             [2, 3, 6, 4, 2, 3]], dtype=np.int64)
         assert_array_equal(actual, desired)
 
+        random = Generator(MT19937(self.seed))
+        actual = random.multinomial([5, 20], [[1 / 6.] * 6] * 2)
+        desired = np.array([[0, 0, 2, 1, 2, 0],
+                            [2, 3, 6, 4, 2, 3]], dtype=np.int64)
+        assert_array_equal(actual, desired)
+
+        random = Generator(MT19937(self.seed))
+        actual = random.multinomial([[5], [20]], [[1 / 6.] * 6] * 2)
+        desired = np.array([[[0, 0, 2, 1, 2, 0],
+                             [0, 0, 2, 1, 1, 1]],
+                            [[4, 2, 3, 3, 5, 3],
+                             [7, 2, 2, 1, 4, 4]]], dtype=np.int64)
+        assert_array_equal(actual, desired)
+
+    @pytest.mark.parametrize("n", [10,
+                                   np.array([10, 10]),
+                                   np.array([[[10]], [[10]]])
+                                   ]
+                             )
+    def test_multinomial_pval_broadcast(self, n):
+        random = Generator(MT19937(self.seed))
+        pvals = np.array([1 / 4] * 4)
+        actual = random.multinomial(n, pvals)
+        assert actual.shape == np.broadcast(n, 1).shape + (4,)
```

Comment:
The significance of that `1` might not be apparent; here and below, I think we can make this check clearer by using the actual shape of `pvals`, by using `broadcast_shapes`:
```suggestion
        assert actual.shape == (np.broadcast_shapes(n.shape, pvals.shape[:-1])
                                + (4,))
```
(The weird line break is for PEP 8.)

Reply:
Done.